docs/spec/api.md

```diff
@@ -117,6 +117,9 @@ specification to correspond with the versions enumerated here.
 			<li>Deleting a manifest by tag has been deprecated.</li>
 			<li>Specified `Docker-Content-Digest` header for appropriate entities.</li>
 			<li>Added error code for unsupported operations.</li>
+			<li>Added capability of doing streaming upload to PATCH blob upload.</li>
```

Comment:
Definitely see this getting confusing if the version number diverges from the Distribution version number. Are you suggesting every API change get a new version number.